### PR TITLE
Relese

### PR DESCRIPTION
**Pull request by YousraMamdouh**

### CODE REVIEW

File: FoodRecipes/Constants.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+//
+//  Constants.swift
+//  FoodRecipes
+//
+//  Created by ME on 26/05/2023.
+//
+
+import Foundation
+struct K{
```
Comment (Ayahamdy97) [inline FoodRecipes/Constants.swift]: Rename struct with meaningful name

File: FoodRecipes/Models/MealModel.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//
+//  MealModel.swift
+//  FoodRecipes
+//
+//  Created by ME on 26/05/2023.
+//
+
+import Foundation
+class MealModel:Codable{
```
Comment (Ayahamdy97) [inline FoodRecipes/Models/MealModel.swift]: Separate classes in different files

File: FoodRecipes/Models/MealModel.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    let count:Int?
+    let results:[Meal]?
+}
+class Meal:Codable{
+    let name:String?
+    let show_id:Int?
```
Comment (Ayahamdy97) [inline FoodRecipes/Models/MealModel.swift]: don't use "_" while naming functions

File: FoodRecipes/Models/MealModel.swift [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    let num_servings:Int?
+
```
Comment (Ayahamdy97) [inline FoodRecipes/Models/MealModel.swift]: remove extra spaces

File: FoodRecipes/Modules/Home/View/CategoriesCell/CategoriesCellTableViewCell.swift [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    @IBOutlet weak var cellView: UIView!
+    @IBOutlet weak var foodRate: UILabel!
+    @IBOutlet weak var foodType: UILabel!
+    @IBOutlet weak var chefName: UILabel!
+    @IBOutlet weak var mealName: UILabel!
+    @IBOutlet weak var favBtn: UIButton!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+         cellView.layer.cornerRadius = 20
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/CategoriesCell/CategoriesCellTableViewCell.swift]: Place code in function

File: FoodRecipes/Modules/Home/View/HomeViewController.swift [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    }
+    
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/HomeViewController.swift]: remove commented code

File: FoodRecipes/Modules/Home/View/HomeViewController.swift [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    let categories: [Category] = [Category(name: "Popular", image: "fire"),Category(name: "Breakfast", image: "breakfast"),Category(name: "Lunch", image: "lunch-box"),Category(name: "Dinner", image: "dinner"),Category(name: "Dessert", image: "cake")]
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        collectionView.register(UINib(nibName: "TopCell", bundle: nil), forCellWithReuseIdentifier: "cell")
+        tableView.register(UINib(nibName: "CategoriesCellTableViewCell", bundle: nil), forCellReuseIdentifier: "CategoriesCellTableViewCell")
+        // Do any additional setup after loading the view.
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/HomeViewController.swift]: dont place code in viewDidLoad, use functions!

File: FoodRecipes/Modules/Home/View/HomeViewController.swift [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+//  HomeViewController.swift
+//  FoodRecipes
+//
+//  Created by Yousra Mamdouh Ali on 26/05/2023.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    let categories: [Category] = [Category(name: "Popular", image: "fire"),Category(name: "Breakfast", image: "breakfast"),Category(name: "Lunch", image: "lunch-box"),Category(name: "Dinner", image: "dinner"),Category(name: "Dessert", image: "cake")]
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/HomeViewController.swift]: place array in view model

File: FoodRecipes/Modules/Home/View/HomeViewController.swift [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+extension HomeViewController: UICollectionViewDelegate, UICollectionViewDataSource ,UICollectionViewDelegateFlowLayout
+{
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return categories.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as! TopCell
+        cell.categoryName.text = categories[indexPath.row].name
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/HomeViewController.swift]: Make function inside cell that setups UI

File: FoodRecipes/Modules/Home/View/TopCustomCell/TopCell.swift [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+//
+
+import UIKit
+
+class TopCell: UICollectionViewCell {
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+        backView.alpha = 1.0
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/TopCustomCell/TopCell.swift:15]: Use Function

File: FoodRecipes/Modules/Home/View/TopCustomCell/TopCell.swift [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+      //  setCellStyle(view: backView)
+    }
+
+    @IBOutlet weak var backView: UIView!
+    @IBOutlet weak var categoryImage: UIImageView!
```
Comment (Ayahamdy97) [inline FoodRecipes/Modules/Home/View/TopCustomCell/TopCell.swift:21]: Place outlets at the beginning of the class

File: FoodRecipes/Services/NetworkManager/Network.swift [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+//  FoodRecipes
+//
+//  Created by ME on 26/05/2023.
+//
+
+import Foundation
+import Alamofire
+
+class Network{
+
```
Comment (Ayahamdy97) [inline FoodRecipes/Services/NetworkManager/Network.swift]: remove extra spaces

File: FoodRecipes/Utilities/Style.swift
Comment (Ayahamdy97) [inline FoodRecipes/Utilities/Style.swift:1]: remove unwanetd comments

File: FoodRecipes/Constants.swift (hunk 1 above)
Comment (mina249) [reply FoodRecipes/Constants.swift]: done

File: FoodRecipes/Models/MealModel.swift (hunk 2 above)
Comment (mina249) [reply FoodRecipes/Models/MealModel.swift]: done

File: FoodRecipes/Models/MealModel.swift (hunk 4 above)
Comment (mina249) [reply FoodRecipes/Models/MealModel.swift]: done

File: FoodRecipes/Services/NetworkManager/Network.swift (hunk 12 above)
Comment (mina249) [reply FoodRecipes/Services/NetworkManager/Network.swift]: done

File: FoodRecipes/Models/MealModel.swift (hunk 3 above)
Comment (mina249) [reply FoodRecipes/Models/MealModel.swift]: done

**Pull request merged**